tools/test/config/feature_compesition/lib1/mbed_lib.json

```diff
@@ -2,7 +2,7 @@
     "name": "lib1",
     "target_overrides": {
         "*": {
-            "target.features_add": ["IPV4"]
```

Comment:
libl?

Also, it has a bootloader?

Reply:
lib1 and lib2 are just arbitrary names for "libraries" in this test. The name of the libraries don't matter and don't affect the contents of the test. They just need to be distinct.

It also doesn't _actually_ have a bootloader, I just needed a feature label that wasn't deprecated. Strangely/happily enough, we don't have that many more deprecated feature labels! I picked the `BOOTLOADER` one since I figured that would be around the longest so I (hopefully) don't have to update these tests in the near future 😄